spec/change_set_persisters/plum_change_set_persister_spec.rb

```diff
@@ -887,4 +887,30 @@
       end
     end
   end
+
+  describe '#save' do
+    context 'when persisting a bag of audiovisual resources in an existing collection' do
+      let(:bag_path) { Rails.root.join('spec', 'fixtures', 'av', 'la_c0652_2017_05_bag') }
+      let(:collection) { FactoryBot.build(:archival_media_collection, source_metadata_identifier: 'C0652') }
+      let(:change_set) { ArchivalMediaCollectionChangeSet.new(collection, source_metadata_identifier: 'C0652', bag_path: bag_path) }
+
+      before do
+        stub_pulfa(pulfa_id: 'C0652')
+        stub_pulfa(pulfa_id: 'C0652_c0377')
+      end
+
+      it 'propagates the imported metadata to all member MediaResources' do
```

Comment:
can you change the text of this spec to reflect that it's pulling metadata from pulfa instead of propagating it